### ISSUE TITLE
MU4 snap release

### ISSUE THREAD

**Issue by Feichtmeier** [musescore/MuseScore#15994]
Please release  musescore4 on the snap for Ubuntu
![grafik](https://user-images.githubusercontent.com/15329494/213800350-3b392fbb-bf52-4500-b3ac-783c98c683df.png)

**Comment by igorkorsukov** [comment]
Unfortunately, we are not engaged in the release of other packages other than those listed in the releases. This is done by third-party people at their discretion

**Comment by Feichtmeier** [comment]
Hi @igorkorsukov thanks for the answer
Are you sure this is not maintained by musescore? The verified flag we receive in our store indicates that this publisher is a verified publisher. In this case it would be a bug then?

https://snapcraft.io/musescore

**Comment by Jojo-Schmitz** [comment]
Yes, we are sure that the snap package is not maintained by the MuseScore in-house team (and only those do make and publish releases)

**Comment by Feichtmeier** [comment]
Alright thanks for clarifying I will forward this then to the snapcrafters team :+1:

**Comment by alexmurray** [comment]
@pachulo are you able to shed any light this? I notice you are the most recent uploader of the musescore snap. Thanks.

**Comment by pachulo** [comment]
Sure. I'd already explained what the status was in [this thread from 2 years ago](https://forum.snapcraft.io/t/call-for-help-maintenance-of-the-musescore-snap/22236), where [in the latest message](https://forum.snapcraft.io/t/call-for-help-maintenance-of-the-musescore-snap/22236/29?u=pachulo) I wrote:
> I think that the best would be for it to be taken over by [@Snapcrafters](https://forum.snapcraft.io/groups/snapcrafters) as upstream is not really interested in having this.

But the main problem here is that, [back in the day](https://forum.snapcraft.io/t/call-for-testing-musescore-v2-1-0/973/32) @thomasbonte verified the snap as coming from Musescore, that's why it shows like that on the store, but the Musescore team was never really interested in maintaining it.

Hope it helps!

**Comment by Jojo-Schmitz** [comment]
> But the main problem here is that, [back in the day](https://forum.snapcraft.io/t/call-for-testing-musescore-v2-1-0/973/32) @thomasbonte verified the snap as coming from Musescore, 

I don't see that.

**Comment by pachulo** [comment]
I think he registered the MuseScore name in the snap store and it was shared with me, so I could upload the snaps:
![MuseScore snap authorization](https://user-images.githubusercontent.com/3256953/214532151-483dfe5d-849d-4e19-8934-17394b7f37f8.png)

Anyway, it's pretty clear that this should either be:
1. Moved to snapcrafters, if they are willing to maintain it
2. or removed from the store.

**Comment by Jojo-Schmitz** [comment]
or
3. taken on by the MuseScoe staff ;-=

**Comment by pachulo** [comment]
> or
> 3. taken on by the MuseScoe staff ;-)

Well, that would be really awesome, as this currently has almost 18k active installs...

**Comment by vpereverzev** [comment]
@Tantacrul I think it worth some time investment to investigate this at least. Shouldn't be too hard to add this configuration to our CI

**Comment by madsrh** [comment]
Also mentioned here https://musescore.org/en/node/343328

Has a solution been found? 🙏 

> that would be really awesome, as this currently has almost 18k active installs...

Yeah, lots of users missing out on all the cool stuff in version 4

**Comment by Jojo-Schmitz** [comment]
This issue got closed, but task #16166 is still open

**Comment by cbjeukendrup** [comment]
I think I'll reopen this and close #16166, because this thread contains some relevant context.

I wondered about https://github.com/musescore/MuseScore/pull/5762: would it make sense to update that PR and merge it eventually? And how much more than that is involved in creating a snap release?

**Comment by Feichtmeier** [comment]
Eventually @kenvandine could help out here?

**Comment by pachulo** [comment]
> I wondered about https://github.com/musescore/MuseScore/pull/5762: would it make sense to update that PR and merge it eventually? And how much more than that is involved in creating a snap release?

That PR contains old code (latest version is https://github.com/pachulo/musescore-snap) to build a snap for MuseScore 3.x, but for versions 4.x and overhaul of it would be required.


> Eventually @kenvandine could help out here?

Maybe the folks from https://github.com/orgs/snapcrafters/people could also help, as they already maintain other QT-based desktop apps...

**Comment by kenvandine** [comment]
@merlijn-sebrechts can you please take a look at updating this?

**Comment by Flimm** [comment]
Could the title of this issue be fixed? The store name is called "Snap Store", not "Ubuntu Store". Snap Store supports multiple distributions, not just Ubuntu.

**Comment by TamaMcGlinn** [comment]
As of february 4th 2026, all snap users of MuseScore 3.6.2 have been force-upgraded to MuseScore 4 because of the way this update was done.

Expect an influx of Ubuntu users commenting on compatibility issues ([32035](https://github.com/musescore/MuseScore/issues/15995) is the one preventing my upgrade). I believe it is important to quickly add a channel for MuseScore3 so that people can downgrade using `sudo snap refresh musescore --channel=3.6.2` or similar. Currently, there is only one channel so there is no choice.

In the meantime, users can do `snap revert musescore`:
```
musescore reverted to 3.6.2
```

(but this only works if you have not already tried reinstalling musescore)

**Comment by TamaMcGlinn** [comment]
If anybody has a way to install the snap package from any other source I would really appreciate instructions. I have several machines with MuseScore 3.6.2 that I could get the snap package from, I just don't know how to get that package onto my main machine, or provide it to others online.

**Comment by Jojo-Schmitz** [comment]
Use the AppImage instead, it is the only supported option

**Comment by TamaMcGlinn** [comment]
I had issues with the appimage with crashes and audio playback not working, which is why I switched to the snap package. The frustrating thing is that the snap package does exist and works perfectly on all my machines, it's just no longer available because it was overwritten on the snap store with musescore4, which has its own new issues.

**Comment by MarcSabatella** [comment]
If you are having an issue of some sort using the current (4.7.2) AppImage, please be sure to report it so it can be investigated and fixed.  Crashes in the AppImage are always fixed if reproducible - that part of what it means to say it is "supported".  For audio issues, it is most likely a simple matter of needing to adjust a setting - best to ask for help on the official Support forum at MuseScore.org

**Comment by Jojo-Schmitz** [comment]
If you're having an issue with old AppImage versions, like the one for 3.6.2, you're out of luck here, it is no longer supported, instead you may want to try my [MuseScore 3.7 Evolution](https://github.com/Jojo-Schmitz/MuseScore/wiki)
For older snap versions go to their support.
